src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -354,14 +370,14 @@ std::unique_ptr<RenderTree> RenderOrchestrator::createRenderTree(const UpdatePar
         }
     }
     // Symbol placement.
+    bool symbolBucketsChanged = false;
     {
         if (!isMapModeContinuous) {
             // TODO: Think about right way for symbol index to handle still rendering
             crossTileSymbolIndex.reset();
         }
 
-        bool symbolBucketsAdded = false;
-        bool symbolBucketsChanged = false;
+        bool symbolBucketsAdded = false;   
         for (auto it = layersNeedPlacement.rbegin(); it != layersNeedPlacement.rend(); ++it) {
             auto result = crossTileSymbolIndex.addLayer(*it, updateParameters.transformState.getLatLng().longitude()); 
             symbolBucketsAdded = symbolBucketsAdded || (result & CrossTileSymbolIndex::AddLayerResult::BucketsAdded);
```

Comment:
Nit: We should probably use `|=` on the next line too.

`symbolBucketsChanged = symbolBucketsChanged || (result != CrossTileSymbolIndex::AddLayerResult::NoChanges);`